server/src/sim/makeworkbook.py

```diff
@@ -602,29 +602,46 @@ def create(self, path):
         self.book.close()
 
 class OptimaGraphTable:
-    def __init__ (self, name, columns, verbose = 2):
+    def __init__ (self, name, sheets, verbose = 2):
         self.name = name
-        self.columns = columns
         self.verbose = verbose
+        self.sheets = sheets
 
     def create(self, path):
```

Comment:
Why do we need and extra function `create` if the only responsibility of the OptimaGraphTable is to produce an excel file? I would suggest we just executed the excel creation in the constructor and pass an extra argument that is the base path where the excel should be stored. The base path should not contain the name, there is already an argument for that